Confus/ConfusServer/Networking/Connection.cpp

```diff
@@ -108,6 +115,15 @@ namespace ConfusServer
 			std::cout << "Message received: " << contents << std::endl;
 		}
 
+        void Connection::sendMessage(RakNet::BitStream& a_InputStream, PacketReliability a_Reliability)
+        {
+            if(m_Connected)
+            {
+                auto openConnections = getOpenConnections();
+                m_Interface->Send(&a_InputStream, PacketPriority::HIGH_PRIORITY, PacketReliability::RELIABLE, 0, openConnections[0], false);
```

Comment:
You are only sending this to the first connection in the list of connections, are you sure that is correct?